etc/omero.properties

```diff
@@ -895,7 +895,7 @@ versions.asm=1.5.3
 versions.axis=1.4
 versions.backport=3.1
 versions.batik=1.8pre-jdk6
-versions.bioformats=5.1.4
```

Comment:
Eventually, yes, but currently the metadata branch is behind. I'm attempting to test openmicroscopy/bioformats#2227 in isolation. With the bump of devspace, metadata-ci hasn't been using the merge build for several days, which means that `bin/omero import -f $TARA` currently lists "UNKNOWN_FORMAT". Once that's working again, I'll look into bumping the bioformats/metadata branch to include 5.1.7 or optimally 5.1.8.